cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
+import re
+
+LOG = logging.getLogger("origin-response")
+
+
+class LambdaClient(object):
+    def __init__(self, conf_file="lambda_conf.json"):
+        self._conf_file = conf_file
+        self._conf = None
+
+    @property
+    def conf(self):
+        if not self._conf:
+            with open(self._conf_file, "r") as json_file:
+                self._conf = json.load(json_file)
+
+        return self._conf
+
+    def handler(self, event, context):
+        # pylint: disable=unused-argument
+
+        request = event["Records"][0]["cf"]["request"]
+        response = event["Records"][0]["cf"]["response"]
+
+        max_age = self.conf["headers"]["max_age"]
+        max_age_pattern_whitelist = [
+            ".+/PULP_MANIFEST",
+            ".+/listing",
+            ".+/repodata/repomd.xml",
+            ".+/ostree/repo/refs/heads/.*/.*",
```

Comment:
I'm a regex newbie -- I don't think this pattern is optimal, but it works. Feel free to suggest a more optimal regex.